[2021-07-17 03:02:58,322] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-07-17T03:02:30.080012+00:00 [queued]>
[2021-07-17 03:02:58,715] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-07-17T03:02:30.080012+00:00 [queued]>
[2021-07-17 03:02:58,717] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-17 03:02:58,717] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-17 03:02:58,718] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-17 03:02:58,848] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_4> on 2021-07-17T03:02:30.080012+00:00
[2021-07-17 03:02:58,850] {standard_task_runner.py:52} INFO - Started process 1263 to run task
[2021-07-17 03:02:58,852] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-07-17T03:02:30.080012+00:00', '--job-id', '13', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp15czzsg0', '--error-file', '/tmp/tmp_2z6xphx']
[2021-07-17 03:02:58,853] {standard_task_runner.py:77} INFO - Job 13: Subtask task_4
[2021-07-17 03:03:00,075] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-07-17T03:02:30.080012+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-17 03:03:01,603] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-07-17T03:02:30.080012+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-17T03:02:30.080012+00:00
[2021-07-17 03:03:01,604] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-17 03:03:01,604] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-07-17 03:03:01,608] {subprocess.py:75} INFO - Output:
[2021-07-17 03:03:04,609] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-17 03:03:05,464] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20210717T030230, start_date=20210717T030258, end_date=20210717T030305
[2021-07-17 03:03:06,167] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-17 03:03:06,197] {local_task_job.py:151} INFO - Task exited with return code 0
